config/optional/config_split.config_split.dev.yml

```diff
@@ -10,7 +10,6 @@ module:
 theme: {  }
 blacklist: {  }
 graylist:
-  - clamav.settings
```

Comment:
Why is this line going?

Reply:
@anthony-malkoun we override dev env by https://github.com/dpc-sdp/content-vic-gov-au/blob/develop/docroot/sites/default/envs/development/settings.php#L25